images/installer/nested-libvirt/README.md

```diff
@@ -0,0 +1,41 @@
+# Nested libvirt on GCE
+
+This image enables launching a libvirt cluster nested in a GCE instance, for libvirt CI testing.
+
+This image contains [nss_wrapper](https://cwrap.org/nss_wrapper.html) to execute ssh commands as
+a mock user to interact with a GCE instance from an OpenShift container.
+
+OpenShift containers run with an arbitrary uid, but ssh requires a valid user.  `nss_wrapper`
```

Comment:
nit "uid" -> "UID" or "user ID"?

And "ssh" -> "`ssh`" (lowercase command) or "SSH" (uppercase protocol acronym)?